experimental/streaming/exporter/reader/format/format.go

```diff
@@ -133,6 +129,31 @@ func AppendEvent(buf *strings.Builder, data reader.Event) {
 	buf.WriteString(" ]\n")
 }
 
+func formatMetricUpdate(buf *strings.Builder, etype observer.EventType, m reader.Measurement) {
+	buf.WriteString(m.Instrument.Kind.String())
+	buf.WriteString(" ")
+	buf.WriteString(m.Instrument.Variable.Name)
+	buf.WriteString("=")
+	buf.WriteString(fmt.Sprint(m.Value))
+	buf.WriteString(" {")
+	i := 0
+	if m.Tags.Len() != 0 {
```

Comment:
`m.Tags.Foreach` should work perfectly fine on an empty map, so this `if` condition can be now removed.

Reply:
(Yeah, it was a nil interface before)